pkgs/by-name/li/linien-gui/package.nix

```diff
@@ -4,16 +4,24 @@
   qt5,
 }:
 
-python3.pkgs.buildPythonApplication rec {
+let
+  python = python3.override {
+    self = python;
+    packageOverrides = self: super: {
+      numpy = super.numpy_1;
+    };
+  };
+in
+python.pkgs.buildPythonApplication rec {
   pname = "linien-gui";
   pyproject = true;
 
-  inherit (python3.pkgs.linien-common) src version;
+  inherit (python.pkgs.linien-common) src version;
 
   sourceRoot = "${src.name}/linien-gui";
 
-  nativeBuildInputs =
-    with python3.pkgs;
+  build-system =
```

Comment:
Sorry, one more thing I overlooked. `wrapQtAppsHook` should be in `nativeBuildInputs`.

Reply:
> Sorry, one more thing I overlooked. `wrapQtAppsHook` should be in `nativeBuildInputs`.

You are correct! Thanks.